packages/mobile/test/schemas.ts

```diff
@@ -1003,6 +1003,7 @@ export const v36Schema = {
   account: {
     ...v35Schema.account,
     finclusiveKycStatus: FinclusiveKycStatus.NotSubmitted,
+    personaKycStatus: undefined,
```

Comment:
shouldn't this be set to whatever `kycStatus` was before? otherwise the migration will result in data loss. This should probably be in a unit test, also

Reply:
oh wait, I thought this was a migrations file

Reply:
I thought about that. The app restarts every time it upgrades right? If so, the kyc value from firebase is fetched every time the app starts, so we shouldn't have anything to worry about

Reply:
don't we need a migration?